src/game/magic/RuneDraw.cpp

```diff
@@ -323,11 +324,17 @@ void ARX_SPELLS_ClearAllSymbolDraw() {
 static void ARX_SPELLS_RequestSymbolDrawCommon(Entity * io, float duration,
                                                RuneInfo & info) {
 	
-	if(!io->symboldraw) {
-		io->symboldraw = new SYMBOL_DRAW;
-	}
+	SYMBOL_DRAW *sd;
 	
-	SYMBOL_DRAW *sd = io->symboldraw;
+	if(io != entities.player()) {
+		if(!io->symboldraw) {
+			io->symboldraw = new SYMBOL_DRAW;
+		}
+		
+		sd = io->symboldraw;
+	} else {
+		sd = &bookSymbolDraw;
+	}
```

Comment:
You can extract a function that takes a `SYMBOL_DRAW *` instead of a `Entity *` so you won't need to special-case the player here.